test/suites/clustering.sh

```diff
@@ -661,7 +661,6 @@ test_clustering_storage() {
     LXD_DIR="${LXD_ONE_DIR}" lxc storage volume copy pool1/vol1 pool1/vol1 --target=node1 --destination-target=node2 --refresh
     LXD_DIR="${LXD_ONE_DIR}" lxc project create foo
     LXD_DIR="${LXD_ONE_DIR}" lxc storage volume copy pool1/vol1 pool1/vol1 --target=node1 --destination-target=node2 --target-project foo
-    LXD_DIR="${LXD_ONE_DIR}" lxc project delete foo
```

Comment:
can we keep this here but change it to a negative to check that the empty check is working for custom volumes please